### PR TITLE
Getmetrics zabbix

### PR DESCRIPTION
**Pull request by zhangzinan**
agent to get metric data from zabbix to insightfinder,use opentsdb api to import data.

### CODE REVIEW

File: zabbix/getmetrics_zabbix.py [hunk 1]
```diff
@@ -0,0 +1,352 @@
...
+            try:
+                res = {}
+                if cachePath and os.path.exists(cachePath):
+                    with open(cachePath, 'r') as f:
+                        res = json.loads(f.read())
+                with open(cachePath, 'w+') as f:
+                    res[hostsHash] = {"items_map": items_map, "items_ids": items_ids}
+                    f.write(json.dumps(res))
+            except:
+                pass
```
Comment (moharnab123saikia) [inline zabbix/getmetrics_zabbix.py]: give a log message instead of just passing the catch

File: zabbix/getmetrics_zabbix.py [hunk 2]
```diff
@@ -0,0 +1,352 @@
...
+    logger.info("Get items from zabbix: " + str(len(items_ids)))
+
+    # get metric data
+    metrics_data = {}
+    chunked_list = chunks(items_ids, parameters['chunkSize'])
+    for sub_list in chunked_list:
```
Comment (moharnab123saikia) [inline zabbix/getmetrics_zabbix.py]: camelcase

File: zabbix/getmetrics_zabbix.py [hunk 3]
```diff
@@ -0,0 +1,352 @@
...
+
+    try:
+        logger.debug("Start to send metric data: {}-{}".format(dataStartTimestamp, dataEndTimestamp))
+        # get instance list from file
+        instanceList = getInstanceListFromFile(agent_config)
+        if len(instanceList) == 0:
+            logger.error("No instances to get data for.")
+            sys.exit()
+
+        chunked_list = chunks(instanceList, parameters['chunkSize'])
```
Comment (moharnab123saikia) [inline zabbix/getmetrics_zabbix.py]: camelcase

**Comment by yqian10** [comment]
Looks good to me. Please follow Morhanab's suggestion to make change.

**Comment by zhangzinan** [comment]
will rewrite it.

**Pull request closed without merging**